sqldelight-compiler/integration-tests/src/test/kotlin/com/example/PlayerQueries.kt

```diff
@@ -126,13 +126,13 @@ class PlayerQueries(private val queryWrapper: QueryWrapper, private val database
     }
 
     fun foreignKeysOn() {
-        val statement = database.prepareStatement(88, "PRAGMA foreign_keys = 1",
+        val statement = database.prepareStatement(88, """PRAGMA foreign_keys = 1""",
                 SqlPreparedStatement.Type.EXECUTE, 0)
         statement.execute()
     }
 
     fun foreignKeysOff() {
-        val statement = database.prepareStatement(89, "PRAGMA foreign_keys = 0",
+        val statement = database.prepareStatement(89, """PRAGMA foreign_keys = 0""",
```

Comment:
:fry_squint:

Reply:
from the kotlinpoet update pr. Forgot to rebuild this file